content/en/xlt/load-testing/Advanced/200-maven-builds.md

```diff
@@ -36,4 +36,33 @@ For versions before XLT 5.0.x, Xceptance offers a public repository hosting all
 
 {{% note title="Version update" %}}
 When configuring your test project to use a newer XLT version, do not forget to update XLT on your load machines as well. The version you’ve used to develop your test scripts must match the execution version of your load test environment.
-{{% /note %}}
\ No newline at end of file
+{{% /note %}}
+
+### Copying Maven Dependencies
+
+If your test suite is using external dependencies/libs, it is required that these are copied into the suite when the project is compiled, because XLT uploads everything to the agent machine but does not compile and resolve there anymore. To automatically copy all non-provided dependencies to `target/dependency` when compiling, add the following snippet to your `pom.xml`:
```

Comment:
Suggestion

If your test suite uses external dependencies or libraries, they must be copied as part of the compile or package step. XLT does not build the project on the agent machines and therefore does not resolve dependencies there. It simply uploads the test suite to the agent, including the contents of the `target` directory. 

To automatically copy all dependencies to `target/dependency` at compile time, add the following snippet to your `pom.xml`: